docs/ivys.js

```diff
@@ -20,6 +20,9 @@ function IvyApp($grid) {
 
 IvyApp.prototype.addItem = function (imageId, imagePath) {
     // adds an image to the masonry grid
+    // Should not be able to add image if it is already on the page
```

Comment:
You could keep an object where the keys are made from the imageId and path to know if it has already been added or not, perhaps. If so, remember to remove it from the object when you remove the image from the page.

I'm not sure I understand how this page "works." What are the pieces? How do they interact? Where is the data? etc